fabtests/unit/mr_test.c

```diff
@@ -178,11 +177,10 @@ static int mr_regattr()
 			base += iov[j].iov_len;
 		}
 
-		ft_fill_mr_attr(&iov[0], NULL, n, ft_info_to_mr_access(fi),
-				FT_MR_KEY, opts.iface, opts.device, &attr, 0);
-		ret = fi_mr_regattr(domain, &attr, 0, &mr);
+		ret = ft_regv_mr(fi, iov, n, ft_info_to_mr_access(fi), FT_MR_KEY,
```

Comment:
ft_regv_mr() would skip the test if "ft_need_mr_reg(fi)" returns false. For the mr tests, we want to always perform the test.

Reply:
> have a way to make ft_need_mr_reg(fi) always true in mr_test?

Either this or have a flag in ft_regv_mr / ft_reg_mr to bypass the check.

Reply:
Can I do the check in ft_reg_mr before calling `ft_regv_mr`, so I don't need to modify the function signatures

Reply:
Taking a second look, it is better not to call ft_*_mr functions for `mr_test`. This is a unit test specifically for testing the individual API functions `fi_mr_reg`, `fi_mr_regv` and `fi_mr_regattr`.

Reply:
OK, I will make mr test handle dmabuf explicitly and call fi_mr_regattr

Reply:
Done